### PR TITLE
Moving from AppEngine SDK to Gcloud SDK

### PR DESCRIPTION
**Pull request by MirkoBonadei**
**Description**
Moving from AppEngine SDK to Gcloud SDK.

**Purpose**
Fixes issue #402 .

### CODE REVIEW

File: build/ensure_gcloud_sdk_is_installed.py [hunk 1]
```diff
@@ -0,0 +1,71 @@
...
+
+
+def main():
+  if not os.path.exists(TEMP_DIR):
+    os.mkdir(TEMP_DIR)
+
+  if os.path.exists(GCLOUD_SDK_PATH):
+    print 'Already has %s, skipping the download' % GCLOUD_SDK_INSTALL_FOLDER
+    _EnsureAppEngineIsInstalled(GCLOUD_SDK_PATH)
+    _Cleanup([os.path.join(TEMP_DIR, GCLOUD_SDK_FILE)])
```
Comment (KaptenJansson) [inline build/ensure_gcloud_sdk_is_installed.py]: Doesn't this remove the SDK?

File: build/ensure_gcloud_sdk_is_installed.py [hunk 2]
```diff
@@ -0,0 +1,71 @@
...
+    return
+
+  _Download(GCLOUD_DOWNLOAD_URL + GCLOUD_SDK_FILE,
+            os.path.join(TEMP_DIR, GCLOUD_SDK_FILE))
+  _Extract(os.path.join(TEMP_DIR, GCLOUD_SDK_FILE), TEMP_DIR)
+  _EnsureAppEngineIsInstalled(GCLOUD_SDK_PATH)
+  _Cleanup([os.path.join(TEMP_DIR, GCLOUD_SDK_FILE)])
```
Comment (KaptenJansson) [inline build/ensure_gcloud_sdk_is_installed.py]: This also removes the SDK, why?

File: README.md [hunk 3]
```diff
@@ -154,7 +154,7 @@ Instructions were performed on Ubuntu 14.04 using Python 2.7.6 and Go 1.6.3.
 
 8\. Build AppRTC using `grunt build` then deploy/run:
 * **If running locally using the Google App Engine dev server (dev/testing purposes)**
-    * Start it using dev appserver provided by the Google app engine SDK `pathToGAESDK/dev_appserver.py  out/app_engine/`.
+    * Start it using dev appserver provided by the Google app engine SDK `pathToGcloudSDK/platform/google_appengine/dev_appserver.py  out/app_engine/`.
```
Comment (KaptenJansson) [inline README.md]: Maybe change Google app engine SDK to Google Cloud SDK?

**Comment by KaptenJansson** [comment]
A general tip is to write "Fixes issue #number" in the description then Github will automatically close the issue once this is merged.

File: README.md (hunk 3 above)
Comment (MirkoBonadei) [inline README.md]: I have left `platform/google_appengine` after `pathToGcloudSDK` beacuse now there are two `dev_appserver.py` (the other one is in `pathToGcloudSDK/bin`).

**Comment by MirkoBonadei** [comment]
PTAL :)

**Pull request merged**